e2e/stack_test.go

```diff
@@ -264,23 +121,35 @@ func ethRollupFlow(t *testing.T, stack *e2e.StackConfig) {
 	balanceBeforeDeposit, err := l1Client.BalanceAt(stack.Ctx, userETHAddress, nil)
 	require.NoError(t, err)
 
+	// wait for two blocks
+	require.NoError(t, stack.WaitL1(2))
```

Comment:
curious - why do you need to wait for two blocks here? Is this to prevent the underflow error you were seeing earlier?